specification/columns/skupricedetails.md

```diff
@@ -1,22 +1,22 @@
 
-The *SkuPriceDetails* column adheres to the following requirements:
+The SkuPriceDetails column adheres to the following requirements:
 
-* The *SkuPriceDetails* column MUST be in [*KeyValueFormat*](#key-valueformat).
+* The SkuPriceDetails column MUST be in [KeyValueFormat](#key-valueformat).
 * The key for a property SHOULD be formatted in [PascalCase](#glossary:pascalcase).
-* The properties (both keys and values) contained in the *SkuPriceDetails* column MUST be shared across all charges having the same *SkuPriceId*, subject to the below provisions.
-  * Additional properties (key-value pairs) MAY be added to *SkuPriceDetails* going forward for a given *SkuPriceId*.
-  * Properties SHOULD NOT be removed from *SkuPriceDetails* for a given *SkuPriceId*, once they have been included.
-  * Individual properties (key-value pairs) SHOULD NOT be modified for a given *SkuPriceId* and SHOULD remain consistent over time.
+* The properties (both keys and values) contained in the SkuPriceDetails column MUST be shared across all charges having the same SkuPriceId, subject to the below provisions.
+  * Additional properties (key-value pairs) MAY be added to SkuPriceDetails going forward for a given SkuPriceId.
+  * Properties SHOULD NOT be removed from SkuPriceDetails for a given SkuPriceId, once they have been included.
+  * Individual properties (key-value pairs) SHOULD NOT be modified for a given SkuPriceId and SHOULD remain consistent over time.
 * The key for a property SHOULD remain consistent across comparable SKUs having that property and the values for this key SHOULD remain in a consistent format.
-* The *SkuPriceDetails* column MUST NOT contain properties which are not applicable to the corresponding *SkuPriceId*.
-* The *SkuPriceDetails* column MAY contain properties which are already captured in other dedicated columns.
-* If a property has a numeric value, it MUST represent the value for a single [*PricingUnit*](#pricingunit).
-* The *SkuPriceDetails* column MUST be present in the billing data when the provider includes a *SkuPriceId*.
-  * The *SkuPriceDetails* column MAY be null when *SkuPriceId* is not null.
-  * The *SkuPriceDetails* column MUST be null when *SkuPriceId* is null.
+* The SkuPriceDetails column MUST NOT contain properties which are not applicable to the corresponding SkuPriceId.
+* The SkuPriceDetails column MAY contain properties which are already captured in other dedicated columns.
+* If a property has a numeric value, it MUST represent the value for a single [PricingUnit](#pricingunit).
+* The SkuPriceDetails column MUST be present in the billing data when the provider includes a SkuPriceId.
```

Comment:
Aligning the terminology with that used in other columns and adding a link to the FOCUS dataset term in the glossary (see PR #592 for linking to the glossary term).

```suggestion
* The SkuPriceDetails column MUST be present in a [*FOCUS dataset*](#glossary:FOCUS-dataset) when the provider includes a SkuPriceId.
```